src/PawsyModules/Meowboard/Meowboard.cs

```diff
@@ -61,6 +61,9 @@ public override void OnDeactivate()
             owner.OnGuildButtonClicked -= ButtonCallback;
             Enabled = false;
         }
+
+        if (!TreasureGame.GameActive || TreasureGame.gameMessage is null) return;
+        TreasureGame.gameMessage.DeleteAsync();
```

Comment:
This fails sometimes because it needs to be awaited or it will just end the closure before it run
`await TreasureGame.gameMessage.DeleteAsync()`